### PR TITLE
Fixed typo quotes and removed password logging

### PR DESCRIPTION
**Pull request by vidhutsingh**
Fixed unnecessary quotes and removed password logging 
Ran the unit test and it went fine. 
Signed-off-by: vidhut-kumar-singh <vidhut-kumar.singh@hpe.com>

### CODE REVIEW

File: pkg/service/active_directory_membership_service.go [hunk 1]
```diff
@@ -119,7 +119,7 @@ func (svc *ActiveDirectoryMembershipService) DeleteActiveDirectoryMembership(id
 func (svc *ActiveDirectoryMembershipService) RemoveActiveDirectoryMembership(id string, user string, password string, force *bool) error {
 
 	if len(id) == 0 || len(user) == 0 || len(password) == 0 {
-		return fmt.Errorf("RemoveActiveDirectoryMembership: invalid parameter specified id: %v, user: %v, password: %v ", id, user, &password)
+		return fmt.Errorf("RemoveActiveDirectoryMembership: invalid parameter specified id: %v, user: %v, ", id, user)
```
Comment (suneeth51) [inline pkg/service/active_directory_membership_service.go:122]: nit: 	Remove the comma/space at the end of the message.

`return fmt.Errorf("RemoveActiveDirectoryMembership: invalid parameter specified id: %v, user: %v", id, user)`

**Review by suneeth51 — APPROVED** [review]
lgtm with a minor comment

File: pkg/service/active_directory_membership_service.go (hunk 1 above)
Comment (vidhutsingh) [reply pkg/service/active_directory_membership_service.go:122]: thanks @suneeth51 .

File: pkg/service/active_directory_membership_service.go [hunk 2]
```diff
@@ -119,7 +119,7 @@ func (svc *ActiveDirectoryMembershipService) DeleteActiveDirectoryMembership(id
 func (svc *ActiveDirectoryMembershipService) RemoveActiveDirectoryMembership(id string, user string, password string, force *bool) error {
 
 	if len(id) == 0 || len(user) == 0 || len(password) == 0 {
```
Comment (rkumpf) [inline pkg/service/active_directory_membership_service.go:121]: An empty password might be a bad idea, but I'm not sure it is invalid.

**Pull request merged**